Projects.md

```diff
@@ -113,6 +113,10 @@ https://github.com/joshuaHallee/foxhole-item-api
 Screenshot to CSV parser    
 https://github.com/miles-igd/foxhole-sheets
 
+#### Foxhole UMAP Textures Extractor:
+Extract all umap textures tiles as PNG
```

Comment:
Please add double space after this line to make sure the url is on the next line. After that I can merge. :)

Reply:
done ;)